integrations/llamaindex/tests/test_toolkit.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+import json
+import logging
+import uuid
+from typing import NamedTuple
+from contextlib import contextmanager
+from unittest import mock
+from pydantic import ValidationError
+
+
+from unitycatalog.ai.client import (
+    set_uc_function_client,
+    get_uc_function_client,
+    FunctionExecutionResult,
+)
+from unitycatalog.ai.databricks import DatabricksFunctionClient
+from unitycatalog_ai_llamaindex.toolkit import LlamaIndexToolkit
+from databricks.sdk.service.catalog import (
+    FunctionInfo,
+    FunctionParameterInfo,
+    FunctionParameterInfos,
+)
+from tests.helper_functions import requires_databricks
+
+CATALOG = "ml"
+SCHEMA = "ben_uc_test"
+_logger = logging.getLogger(__name__)
+
+
+class FunctionObj(NamedTuple):
+    full_function_name: str
+    comment: str
+
+
+@contextmanager
+def create_function_and_cleanup(client: DatabricksFunctionClient):
+    func_name = f"{CATALOG}.{SCHEMA}.test_{uuid.uuid4().hex[:4]}"
+    comment = "Executes Python code and returns its stdout."
+    sql_body = f"""CREATE OR REPLACE FUNCTION {func_name}(code STRING COMMENT 'Python code to execute. Remember to print the final result to stdout.')
+RETURNS STRING
+LANGUAGE PYTHON
+COMMENT '{comment}'
+AS $$
+    import sys
+    from io import StringIO
+    stdout = StringIO()
+    sys.stdout = stdout
+    exec(code)
+    return stdout.getvalue()
+$$
+"""
+    try:
+        client.create_function(sql_function_body=sql_body)
+        yield FunctionObj(full_function_name=func_name, comment=comment)
+    finally:
+        try:
+            client.client.functions.delete(func_name)
+        except Exception as e:
+            _logger.warning(f"Failed to delete function: {e}")
+
+
+@pytest.fixture
+def client():
```

Comment:
Master branch has supported serverless, could you update the test to include serverless client as well? I've updated langchain tests :)

Reply:
thanks for the heads up! Updated :)